<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>szukajObslugi</class>
 <widget class="QDialog" name="szukajObslugi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="szukajButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrocButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QCheckBox" name="nazwiskoCB">
      <property name="text">
       <string>Nazwisko</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="imieLE"/>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="rejonCB">
      <property name="text">
       <string>Rejon</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="nazwiskoLE"/>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="imieCB">
      <property name="text">
       <string>Imie</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="rejonLE"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
